shaders/lang/ru_RU.lang

```diff
@@ -61,684 +120,1432 @@ screen.Direct_Light = Прямой свет
-
-
-screen.Subsurface_Scattering = Поверхностное рассеивание
-    option.LabSSS_Curve = Интенсивность LabSSS
-    option.MISC_BLOCK_SSS = SSS для случайных блоков
-    option.MOB_SSS = SSS для мобов
-    option.Ambient_SSS = SSS для неба
-    option.ambientsss_brightness = SSS для неба
+            value.shadowDistance.768.0 = 48 чанков
+            value.shadowDistance.1024.0 = 64 Чанков
+            value.shadowDistance.1536.0 = 96 Чанков
+            value.shadowDistance.2048.0 = 128 Чанков
+            value.shadowDistance.4096.0 = 256 Чанков
+            value.shadowDistance.8192.0 = 512 Чанков
+
+[-----Max Shadow Bounds-----]
+
+        option.OPTIMIZED_SHADOW_DISTANCE = Максимальные Грани Теней
+            value.OPTIMIZED_SHADOW_DISTANCE.-1.0 = Не-Оптимизированные
+            value.OPTIMIZED_SHADOW_DISTANCE.1.0 = Оптимизированные
+
+[-----Entity Shadow Distance-----]
+
+        option.entityShadowDistanceMul = Дистанция Теней Сущности
+            value.entityShadowDistanceMul.0.01 = 1% от Дистанции Теней
+            value.entityShadowDistanceMul.0.02 = 2% от Дистанции Теней
+            value.entityShadowDistanceMul.0.03 = 3% от Дистанции Теней
+            value.entityShadowDistanceMul.0.04 = 4% от Дистанции Теней
+            value.entityShadowDistanceMul.0.05 = 5% от Дистанции Теней
+            value.entityShadowDistanceMul.0.10 = 10% от Дистанции Теней
+            value.entityShadowDistanceMul.0.15 = 15% от Дистанции Теней
+            value.entityShadowDistanceMul.0.20 = 20% от Дистанции Теней
+            value.entityShadowDistanceMul.0.25 = 25% от Дистанции Теней
+            value.entityShadowDistanceMul.0.30 = 30% от Дистанции Теней
+            value.entityShadowDistanceMul.0.35 = 35% от Дистанции Теней
+            value.entityShadowDistanceMul.0.40 = 40% от Дистанции Теней
+            value.entityShadowDistanceMul.0.45 = 45% от Дистанции Теней
+            value.entityShadowDistanceMul.0.50 = 50% от Дистанции Теней
+            value.entityShadowDistanceMul.0.75 = 75% от Дистанции Теней
+            value.entityShadowDistanceMul.1.00 = 100% от Дистанции Теней
+
+|/////Shadow Filtering Settings/////|
+
+    screen.Filtering = Настройка Фильтра Теней
+
+[-----Basic Shadow Filter Samples-----]
+
+        option.SHADOW_FILTER_SAMPLE_COUNT = Обычный Образец Фильтра Теней
+
+[-----Minimum Shadow filter Radius-----]
+
+        option.Min_Shadow_Filter_Radius = Минимальный Радиус Фильтра Теней
+
+[-----Variable Penumbra Shadows-----]
+
+        option.Variable_Penumbra_Shadows = Переменные Полутеневые Тени
+
+[-----VPS Filter Samples-----]
+
+        option.VPS_Search_Samples = Образцы Фильтра ППТ (VPS)
+
+[-----Maximum Shadow Filter Radius-----]
+
+        option.Max_Shadow_Filter_Radius = Максимальный Радиус Фильтра Теней
+
+[-----Sun Size Factor-----]
+
+        option.Max_Filter_Depth = Фактор Размера Солнца
+
+|/////FloodFill/////|
+
+    screen.LPV = НаливноеЗаполнение (FloodFill) - §cОБЯЗАТЕЛЬНО IRIS§r
```

Comment:
```suggestion
    screen.LPV = FloodFill - §cОБЯЗАТЕЛЕН IRIS§r
```

Reply:
Deleting the FloodFill translation is weird.

You could have deleted the whole thing then.
But you probably don't know the correct translation of FloodFill.

I don't think it's that important because they translate the same in English (REQUIRED IRIS).